README.md

```diff
@@ -1,3 +1,51 @@
+
+## RESTy Phase 1:
+
+Begin work on the RESTy API testing application
+
+Phase 1 Requirements
+Today, we begin the first of a 4-Phase build of the RESTy application, written in React. In this first phase, our goal is to setup the basic scaffolding of the application, with intent being to add more functionality to the system as we go. This initial build sets up the file structure so that we can progressively build this application in a scalable manner
+
+The following user/developer stories detail the major functionality for this phase of the project.
+
+- As a user, I expect an easy to read and understand user interface so that I can use the application intuitively
+
+- As a user, I want to enter the URL to a REST API and select the REST method to use to access it
+
+- As a user, I want visual confirmation that my entries and selections are valid so that I have confidence the application will be able to fetch the API data that I’ve requested
+
+And as developers, here are the high level development tasks that address the above end user requirements
+
+- Create a visually appealing site with a Header, Footer, and a large content area
+
+- Create a form that asks for a URL
+
+- Create buttons that let the user choose from the REST methods (get, post, put, delete)
+
+When the form is filled out, and the button is clicked, display the URL and the method chosen
+
+
+# Resources
+- Collaborated with: Jason Quaglia, Carly Dekock and Jason Dormier
+- Also adapted from William Moreno who's code was used for lab review today
+
+# Netlify Deploy
+
+[https://resty2.netlify.app/](https://resty2.netlify.app/)
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/68fba5fb-500a-4ad0-8f50-11c0f450fcd1/deploy-status)](https://app.netlify.com/sites/resty2/deploys)
+
+
+
+
+
+
+
+
+
+
+<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/>
```

Comment:
- Would like to see a help and history tab added to this
- Would like to see this project deployed, current link doesn't work
- Would like to see some styling added as well